mozci/console/commands/push.py

```diff
@@ -407,6 +407,9 @@ def handle(self) -> None:
                 self.line(f"<error>{traceback.format_exc()}</error>")
                 continue
 
+            # Rettriger test failures to detect permanent ones, only on Linux cloud worker pool for now
```

Comment:
Do you think it is ok to run the identification at the push level, right after calling `classify` ?

Reply:
Yes, good to run it at the push level.